src/geometry/WorkingGeometry.ts

```diff
@@ -279,5 +261,13 @@ export class WorkingGeometry {
         this.vertices = this.vertices.map((workingVec: vec4) =>
```

Comment:
Nit: can we capitalize "transformation matrix" in the docstring above?

Reply:
Do you mean in the error? I don't see a docstring that has it. If that's what you mean, I'm OK with changing that (provided we use the same convention with other error strings).

Reply:
I can't comment on that line since it was from an old PR and I just saw it while browsing this one. It's on like 258 (formerly 276) of this file. I'm ok w/ not changing it, and I'll just follow up with it in a different PR.